Client/Frontend/Login Management/LoginDetailTableViewCell.swift

```diff
@@ -29,21 +29,26 @@ enum LoginTableViewCellStyle {
 
 class LoginDetailTableViewCell: ThemedTableViewCell {
 
-    fileprivate let labelContainer = UIView()
+    fileprivate lazy var labelContainer: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
```

Comment:
`translatesAutoresizingMaskIntoConstraints` can be done implicitly by using .build 
```suggestion
    private lazy var labelContainer: UIView = .build { _ in }
```

I won't make the comments everywhere in the PR, but basically everywhere you added the line `.translatesAutoresizingMaskIntoConstraints = false` you can use `.build` instead.